build/tools/templates/SharedFxSymbols/Microsoft.AspNetCore.All.symbols.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.AspNetCore.All.symbols</id>
+    <version>$VERSION$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
+    <projectUrl>http://www.asp.net/</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <description>Symbol packages for Microsoft.AspNetCore.All shared framework</description>
+    <copyright>© .NET Foundation. All rights reserved.</copyright>
```

Comment:
Can you update copyright and licenseUrl to match our runtime packages? 

https://github.com/aspnet/BuildTools/blob/dev/src/Internal.AspNetCore.Sdk/build/Common.props#L13-L22

I'd actually prefer to flow these automatically from Internal.AspNetCore.Sdk so we don't have to remember to update more places.

Reply:
How do we automatically flow these from Internal.AspNetCore.Sdk? Add them all to properties passed into PackNuspec?

Reply:
Build the symbols nupkg from a MSBuild project that references Internal.AspNetCore.Sdk.